lib/http_perform.c

```diff
@@ -12,6 +12,7 @@
 #endif
 
 #include <string.h>
+#include <unistd.h>
```

Comment:
Header not available on windows
#include <windows.h> this header in windows has Sleep() API defined.